schemas/com.snowplowanalytics.snowplow/media_player_event_user_update_quality/jsonschema/1-0-0

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Media player event tracked when the video playback quality changes as a result of user interaction (choosing a different quality setting).",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "media_player_event_user_update_quality",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
```

Comment:
`quality_selected` and maybe `quality_options` to indicate what selections are available to the user.

Reply:
Thinking more about this, do we actually need this schema? My intention was just to differentiate between the quality being changed automatically or by user. But this can actually be handled by an `automatic` boolean property in the `quality_change` schema as you suggested. So unless we really want to capture the quality options offered to users, I am thinking of dropping this schema.